src/main/assembly/fakeroot.xml

```diff
@@ -103,6 +103,12 @@ Installed in a fakeroot directory structure.
             <outputDirectory>/etc/init.d</outputDirectory>
             <fileMode>0755</fileMode>
         </fileSet>
+	<!-- .info files -->
+        <fileSet>
+            <directory>src/main/infofiles</directory>
+            <outputDirectory>/etc/grid-security/certificates</outputDirectory>
```

Comment:
Since these files are fully argus-specific, then it's up to the PIP to look them up in whatever directory we choose, or am I missing something? I would go for something into /etc/argus/, but also /etc/grid-security/argus/ would be fine for me (following an approach similar to /etc/grid-security/vomsdir).